.pre-commit-config.yaml

```diff
@@ -40,7 +40,7 @@ repos:
             entry: pyupgrade
             language: system
             types: [python]
-            args: [--py38-plus]
+            args: [--py38-plus, --keep-runtime-typing]
```

Comment:
likely needs upgrade to --py39-plus since we do not support python 3.8 any longer